src/app.js

```diff
@@ -1,10 +1,29 @@
 /**
+ * @member {Object} books - declaration the new instance of collection Books
+ * which contains the last 10 books from the query
  * @member {Object} library - declaration the new instance of collection
+ * which contains all books from the query
  */
 Collections = {
-    books: new Books()
+    books: new Books(),
+    library: new Library()
```

Comment:
For what do you need library collection ?

Reply:
The librari collection is needed to store earlier requests.

Reply:
Librari collection is needed to store earlier requests